.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,22 +1,48 @@
 ---
-name: Bug Report
-about: Reporting issues with the VA API Platform
-
+name: 🔴 Bug Report
+about: For reporting issues with the VA API Platform
 ---
 
-## APIs Affected
+Important note: As this is a public forum, please do not include your VA API OAuth Client ID, VA API OAuth Client Secret, VA API key, or any Personally Identifiable Information (PII) or Protected Health Information (PHI) in your report.
+
+### Troubleshooting note
```

Comment:
🤔 not sure this needs to be separated out from the note above. Maybe combine the two as a "Please read before submitting".

Reply:
Done!